yyu/identity-concentrator/kustomization.yaml

```
@@ -0,0 +1,25 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+  env: yyu
+namespace: yyu
+patchesStrategicMerge:
+  - istio-ingress.yaml
+  - |-
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: ic-deployment
+    spec:
+      template:
+        spec:
+          containers:
+          - name: ic
+            env:
+            - name: HYDRA_INTERNAL
+              value: hydra-ic-service.yyu.svc.cluster.local
+resources:
+  # Will get overwritten with correct version on deploy:
+  - github.com/databiosphere/identity-concentrator//config
```

```suggestion
  - github.com/databiosphere/identity-concentrator//config?ref=0.0.0
```
This will let kustomize make the correct base kustomization version adjustment:
https://github.com/broadinstitute/github-action-kustomize-deploy/blob/1d01daec2b7ede60bd2c75f368ed6fb5d1e2a86a/apply.sh#L93